spec/models/core_induction_programme.rb

```diff
@@ -8,4 +8,15 @@
     it { is_expected.to have_many(:early_career_teacher_profiles) }
     it { is_expected.to have_many(:early_career_teachers).through(:early_career_teacher_profiles) }
   end
+
+  describe "course_years" do
+    it "returns multiple course years for a single core_induction_programme" do
+      core_induction_programme = FactoryBot.create(:core_induction_programme)
+      course_year_one = FactoryBot.create(:course_year, core_induction_programme: core_induction_programme)
+      course_year_two = FactoryBot.create(:course_year, core_induction_programme: core_induction_programme)
+
+      expect(course_year_one[:core_induction_programme]).to eql(course_year_two[:core_induction_programme])
```

Comment:
`course_year_one.core_induction_programme`? I don't see why we would access it like a hash.

Reply:
My bad, that's me forgetting we can access with that because of associations